src/subcommand/add.rs

```diff
@@ -16,13 +17,28 @@ impl Add {
         let mut db = util::get_db()?;
         let now = util::get_current_time()?;
         let maxage = config::zo_maxage()?;
+        let exclude_dirs = config::zo_exclude_dirs();
 
         match &self.path {
```

Comment:
To avoid code duplication, I think it would be a lot better if we did something like this:

```rust
let path = match &self.path {
    Some(path) => path.clone(),
    None => env::current_dir().context("unable to fetch current directory")?,
};
```

Also, do change the type of `Add::path` to `PathBuf`.